<launch>
  <node pkg="ros_ht_msg" type="ros_ht_msg" name="publish_ht_msg" output="screen">
  </node>
  <node pkg="ros_ht_msg" type="ht_move.py" name="pub_command2" output="screen">
  </node>
</launch>
